src/SearchButton.js

```diff
@@ -0,0 +1,7 @@
+import React from "react";
```

Comment:
Excellent job extracting the search button. Good choice of name for your component and good to see you use PascalCase for your component and file names.

I see you've used the function syntax to create this `SearchButton` component. If you wanted to do this using [arrow functions](https://syllabus.codeyourfuture.io/react/week-1/lesson#arrow-functions-for-shorter-syntax) what would the syntax look like?

Reply:
Noted.